code/modules/mob/living/carbon/human/blood.dm

```diff
@@ -0,0 +1,309 @@
+/****************************************************
+				BLOOD SYSTEM
+****************************************************/
+//Blood levels
+var/const/BLOOD_VOLUME_SAFE = 501
+var/const/BLOOD_VOLUME_OKAY = 336
+var/const/BLOOD_VOLUME_BAD = 224
+var/const/BLOOD_VOLUME_SURVIVE = 122
+
+/mob/living/carbon/human/var/datum/reagents/vessel	//Container for blood and BLOOD ONLY. Do not transfer other chems here.
```

Comment:
Of all the things I could comment about... I think this should be renamed to "circulatory", or something like that.